CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## [Unreleased]
 ### Changed
+- [#22](https://github.com/CallistoLabsNYC/samsa/issues/22)Include docker-compose stack in the CICD
```

Comment:
Suggest this is kept sorted descending. And move #19 from Changed to a Fixed section.